sda-commons-server-weld-example/src/test/java/org/sdase/commons/server/weld/WeldJunit5AutoExtensionTest.java

```diff
@@ -0,0 +1,20 @@
+package org.sdase.commons.server.weld;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.jboss.weld.junit5.auto.AddBeanClasses;
+import org.jboss.weld.junit5.auto.EnableAutoWeld;
+import org.junit.jupiter.api.Test;
+import org.sdase.commons.server.weld.beans.ExampleProducer;
+import org.sdase.commons.server.weld.beans.UsageBean;
+
+@EnableAutoWeld
```

Comment:
With these new extensions, I would also deprecate our custom implementation and mention the new options in the migration guide. In v7 we can then remove our implementation.